### ISSUE TITLE
MA0074 Incorrectly Links MA0001 in Intellisense

### ISSUE THREAD

**Issue by aolszowka** [meziantou/Meziantou.Analyzer#198]
**Context**
Rule identifier: _MA0074_
Target Framework: _.NET 3.1_
Language version: _9.0_

**Describe the bug**
The following code snippet correctly throws MA0074, however the documentation link incorrectly links to MA0001.

**To Reproduce**

````c#
"Hello World!".IndexOf("World");
````

![image](https://user-images.githubusercontent.com/2592881/97231812-b9d74b00-17a1-11eb-80bf-91135bceae33.png)

**Additional context**
It is unclear how many others are affected, it would be worth a spin through of existing code paths.

Thank you for this Analyzer I am going to continue to suggest it as we get closer to .NET 5 when a bunch of people are going to wonder why their string comparisons start falling over for "no reason".

**Comment by meziantou** [comment]
Three rules have the wrong help link (copy/paste errors...). I've added a check and fixed the three links in #199.

Thanks for your contribution 😊
